requirements/common_constraints.txt

```diff
@@ -0,0 +1,48 @@
+# This is a temporary solution to override the real common_constraints.txt
```

Comment:
Maybe none of these constraints matter, except removing Django, so it doesn't matter that this is so out of date with https://github.com/edx/edx-lint/blob/master/edx_lint/files/common_constraints.txt? But, it seems like it would be nice to have these match (outside of removing Django).

Reply:
the file has been updated